site/src/pages/components/menu.explainer.mdx

```diff
@@ -226,11 +234,11 @@ When user clicks on a menuitem:
     2. Else, open it and move focus.
 2. Else, if menuitem is contained inside an open menulist, close its parentMenulist.
 
-The algorithm when user clicks on a menuitemcheckbox:
+The algorithm when user clicks on a checkbox menuitem:
 
 1. Set it to checked.
 
-The algorithm when user clicks on a menuitemradio:
+The algorithm when user clicks on a radio menuitem:
 
 1. Set it to checked.
 2. Set all other menuitemradio to unchecked.
```

Comment:
was thinking this should be clearer that it's not _all_ other radio menuitems - but rather any previously checked menuitem radio within that specific group.  

maybe this line can be left alone if line 241 said "The algorithm when a user activates a menuitem within a `checkable=multiple` fieldset" ?